src/dist/replication/lib/replica_stub.cpp

```diff
@@ -170,6 +172,10 @@ void replica_stub::initialize(const replication_options& opts, bool clear/* = fa
         dir_list.insert(dir_list.end(), tmp_list.begin(), tmp_list.end());
     }
 
+    replicas rps;
+    utils::ex_lock rps_lock;
```

Comment:
We should use service::zlock instead to enable tooling unless it is really performance critical for the lock overhead.